docs/ibm-cloud-rules.md

```diff
@@ -2224,6 +2231,81 @@ paths:
 </table>
 
 
+### Rule: inline-property-schema
+<table>
+<tr>
+<td><b>Rule id:</b></td>
+<td><b>inline-property-schema</b></td>
+</tr>
+<tr>
+<td valign=top><b>Description:</b></td>
+<td>If a nested schema (perhaps a schema property, an array <code>items</code> schema,
+an <code>additionalProperties</code> schema, etc.) is an object with user-defined properties,
+it is a best practice to define the schema as a named schema within the <code>components.schemas</code> section
+of the API definition, and then reference it with a schema $ref instead of defining it as an inline object schema.
+This is a best practice because the SDK generator will use the schema reference when determining the datatype associated
```

Comment:
I think we (when relevant) include Handbook references also, right? I think this is directly applicable here: https://test.cloud.ibm.com/docs/api-handbook?topic=api-handbook-schemas#nested-object-schemas

Reply:
updated it